Core/include/EventTuple.h

```diff
@@ -159,6 +160,14 @@ using MetCovMatrix = analysis::SquareMatrix<2>;
     VAR(Float_t, lhe_H_m) \
     VAR(Float_t, lhe_hh_m) \
     VAR(Float_t, lhe_hh_cosTheta) \
+    VAR(UInt_t, lhe_index) \
+    VAR(UInt_t, lhe_pdgId) \
+    VAR(UInt_t, lhe_mother_index) \
+    VAR(LorentzVectorM, lhe_p4) \
```

Comment:
- index, pdgId, mother_index and p4 must be vectors.
- use int for the index and pdg id